src/main/kotlin/no/nav/familie/ef/mottak/repository/DbMigreringRepository.kt

```diff
@@ -12,7 +12,7 @@ import org.springframework.transaction.annotation.Transactional
 interface DbMigreringRepository : JpaRepository<Task, Long> {
 
     @Modifying
-    @Query("update Task t set t.taskStepType = :newType where t.taskStepType = :oldType")
+    @Query("update Task t set t.type = :newType where t.type = :oldType")
```

Comment:
Må vi gjøre endringer i migreringsfiler også? Eller er det bare "magien" som funker?

Reply:
Er dette noe vi har valgt å gjøre nå, eller som vi _må_ gjøre for å ta i bruk prosessering-backend?

Reply:
Feltet heter type i databasen, men feltet i felles-data-class hete taskStepType, den renamet @bent-lorentzen til type